k8s/aat/common/idam/idam-api.yaml

```diff
@@ -15,13 +15,20 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-api
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:aat-16d37681
-      ingressHost: idam-api.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/api:prod
+      replicas: 1
```

Comment:
why only 1 replica?

Reply:
We have a problem to solve of pod level session affinity. Until then, 1 replica.

Reply:
and it's not a problem with multiple clusters? as even though you're setting 1 here there will be another cluster running with it